src/index.ts

```
@@ -23,47 +27,60 @@ app.use("/", indexRouter);
 app.listen(PORT);
 
 // Workaround: Hide DEBUG logs from appearing in console
-console.debug = (): void => {};
+console.debug = _.noop;
 
 configure({
     appenders: {
         out: { type: "stdout" },
         file: { type: "file", filename: "debug.log" },
     },
-    categories: { default: { appenders: ["file", "out"], level: "debug" } },
+    categories: { default: { appenders: ["file", "out"], level: development ? "all" : "debug" } },
 });
 
 // Root folder on "yarn start" is ./src, ask path to go back one level
-const rootFolder = process.env.NODE_ENV === "development" ? ".." : "";
-const { config } = yargs.options({
-    config: {
-        type: "string",
-        alias: "c",
-        describe: "Configuration file",
-        default: path.join(__dirname, rootFolder, "app-config.json"),
-    },
-}).coerce("config", path => {
-    if (fs.existsSync(path)) {
-        return JSON.parse(fs.readFileSync(path, "utf8"));
-    } else {
-        throw new Error("Configuration file not found");
+const rootFolder = development ? ".." : "";
+const { config } = yargs
+    .options({
+        config: {
+            type: "string",
+            alias: "c",
+            describe: "Configuration file",
+            default: path.join(__dirname, rootFolder, "app-config.json"),
+        },
+    })
+    .coerce("config", path => {
+        if (fs.existsSync(path)) {
```

Minor: Cant we simply run read+parse and let the functions throw the exceptions?